modules/developer-cli-odo-installing-odo-on-linux.adoc

```diff
@@ -6,88 +6,40 @@
 
 = Installing {odo-title} on Linux
 
-== Binary installation
+The `{odo-title}` CLI is available to download as a binary and as a tarball for multiple operating systems and architectures including: 
 
-.Procedure
-
-. Obtain the binary:
-+
-[source,terminal]
-----
-# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64 -o /usr/local/bin/odo
-----
-
-. Change the permissions on the file:
-+
-[source,terminal]
-----
-# chmod +x /usr/local/bin/odo
-----
+[cols="2,1,1",options="header"]
+|===
+|Operating System|Binary|Tarball
+|Linux|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64[odo-linux-amd64] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz[odo-linux-amd64.tar.gz]
+|Linux on IBM Power|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le[odo-linux-ppc64le] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le.tar.gz[odo-linux-ppc64le.tar.gz]
+|Linux on IBM Z and LinuxONE|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x[odo-linux-s390x] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x.tar.gz[odo-linux-s390x.tar.gz]
+|===
 
-== Tarball installation
 
 .Procedure
 
-. Obtain the tarball:
+. Navigate to link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[] and download the appropriate file for your operating system and architecture.
+. If you downloaded a tarball, extract the binary:
 +
 [source,terminal]
 ----
-# sh -c 'curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz | gzip -d > /usr/local/bin/odo'
+$ tar xvzf <filename>
 ----
-
-. Change the permissions on the file:
+. Change the permissions on the binary:
 +
 [source,terminal]
 ----
-# chmod +x /usr/local/bin/odo
+$ chmod +x <filename>
 ----
-
-== Installing with yum on {op-system-base-full}
-
-.Procedure
-
-. Register with Red Hat Subscription Manager:
+. Place the `{odo-title}` binary in a directory that is on your `PATH` and rename it to `odo`, if necessary.
```

Comment:
I would eliminate the `if necessary` users can make that choice. 

`Place the odo binary in a directory that is on your PATH and rename it to odo.`

Reply:
I have reworked this so that the renaming is done earlier in the procedure.

I have copied the pattern for checking the PATH from the oc docs, so they are consistent with those.